default/stringtables/en.txt

```diff
@@ -5473,6 +5499,32 @@ Repairs are delayed for one turn for ships just entering the system. Ships need
 
 Any combat in the system will disrupt repairs for that turn.'''
 
+CONFIG_GUIDE_TITLE
+Configuration Guide
+
+CONFIG_GUIDE_TEXT
+'''Most configuration options for the game are stored and read from a single XML formatted file named config.xml.
+If this file does not exist (or is for a different build or version) it will be (re)created when the client starts. 
+When an option is changed, the configuration file is updated.
+Some of these options can be changed from the [[OPTIONS_TITLE]] menu, under the Main Menu.
+
+An option set in the persistent config file (if one exists) will override the value from config.xml.
+An option set from a command line argument will override the value from both files.
```

Comment:
Clarify: will setting an option on the command line result in it being written to config.xml ?

Reply:
Added a sentence to this effect.